components/mediation-ui/org.wso2.carbon.proxyadmin.ui/src/main/resources/web/proxyservices/index.jsp

```diff
@@ -579,13 +580,20 @@
                     } else if (wsdlUri.value == '') {
                         return proxyi18n["invalid.wsdl.uri"];                        
                     }
-                } else if (mode == 'reg') {
+                }
+
+                else if (mode == 'reg') {
                     var wsdlKey = getElement('wsdlRegText');
                     if (wsdlKey && wsdlKey.value == '') {
                         return proxyi18n['wsdl.not.selected'];
                     }
                 }
             }
+            var serviceGroupElm = getElement("serviceGroup");
+            if(serviceGroupElm.value != null && serviceGroupRegex.test(serviceGroupElm.value)) {
```

Comment:
Check the possibility of serviceGroupElm variable being null. If possible, do null check as in line 587